index.html

```diff
@@ -17,5 +17,71 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+
+    <!-- Header of the website that includes logo and navigation -->
+     <header class="header-container"> 
+        <div class="header-inner">
+            <img class="logo" src="img/karma-logo.svg" alt="Logo">
+            <nav>
+            <ul class="navigation">
+                <li><a class="karma-text" href="default.asp">Meet Karma</a></li>
+                <li><a href="howitworks.asp">How it Works</a></li>
+                <li><a href="store.asp">Store</a></li>
+                <li><a href="blog.asp">Blog</a></li>
+                <li><a href="help.asp">Help</a></li>
+                <li><a href="login.asp">Login</a></li>
+              </ul>   
+            </nav>
+        </div>
+    </header>
+
+
+    <!-- Main content of the website -->
+    <main>
+        <div id="hero-container">
```

Comment:
No IDs please if it's just for styling. 
IDs can be useful for Javascript, but avoid for CSS.